src/components/sidebar/index.tsx

```diff
@@ -104,11 +104,25 @@ const Sidebar = ({
 
 	let sidebarContent
 	const filteredMenuItems = getFilteredNavItems(itemsWithMetadata, filterValue)
-	const navResourceItems = generateResourcesNavItems(currentProduct?.slug).map(
-		(item) => item
+	const navResourceItems = generateResourcesNavItems(currentProduct?.slug)
```

Comment:
the returned value of `generateResourcesNavItems()` is an array, so using `.map` here is redundant.